### PR TITLE
chore: first pass at a PF4 based layout

### PR DESCRIPTION
**Pull request by riccardo-forina**
pf4-layout

### CODE REVIEW

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1019/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by gashcrumb** [comment]
Huge improvement on most pages!  The manage CI/CD page and import integration pages both need some treatment though:

image

image

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1030/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by seanforyou23** [comment]
Nice work @riccardo-forina! Looks like you've addressed a good bit of what was described here https://github.com/syndesisio/syndesis/issues/5293

Is there bandwidth to also move the tabs from the customizations page into proper expandable nav items? That would go a long way toward standardizing the navigation experience.

**Comment by riccardo-forina** [comment]
Think so, lemme check.

**Comment by gashcrumb** [comment]
ooh yeah if we could get the nav links in the sidebar that'd be awesome, doesn't have to be this PR though.

**Comment by riccardo-forina** [comment]
Too late doing that 😂

**Comment by gashcrumb** [comment]
I think you're just looking for bonus :beers:

**Comment by riccardo-forina** [comment]
Right, this should do it. The 'Customizations' menu on a refresh of any of its subpages will be rendered closed. And refreshing the api connector page is crapping out with a terrible error I don't care much about right now :)

File: app/ui-react/packages/ui/src/Integration/IntegrationEditorLayout.tsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 // tslint:disable react-unused-props-and-state
 // remove the above line after this goes GA https://github.com/Microsoft/tslint-microsoft-contrib/pull/824
```
Comment (seanforyou23) [inline app/ui-react/packages/ui/src/Integration/IntegrationEditorLayout.tsx]: Think these can be removed now?

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1048/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by gashcrumb** [comment]
:beers: :beers: :beers: :beers: :beers: :beers: :beers: :beers:

**Pull request merged**